editions/tw5.com/tiddlers/nodejs/tiddlywiki.files_Files.tid

```diff
@@ -48,13 +48,14 @@ Directory specifications in the `directories` array may take the following forms
 
 * a ''string'' literal, specifying the absolute or relative path to the directory containing the tiddler files (relative paths are interpreted relative to the path of the `tiddlywiki.files` file). The directory is recursively searched for tiddler files
 * <<.from-version "5.1.14">> an ''object'' with the following properties:
-** ''path'' - (required) the absolute or relative path to the directory containing the tiddler files (relative paths are interpreted relative to the path of the `tiddlywiki.files` file). Note that the directory is not recursively searched; sub-directories are ignored
+** ''path'' - (required) the absolute or relative path to the directory containing the tiddler files (relative paths are interpreted relative to the path of the `tiddlywiki.files` file). Note that by default the directory is not recursively searched; sub-directories are ignored unless the //searchSubdirectories// flag is set to `true` (see below).
 ** ''filesRegExp'' - (optional) a [[regular expression|https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Regular_Expressions]] that matches the filenames of the files that should be processed within the directory
 ** ''isTiddlerFile'' - (required) if `true`, the file will be treated as a [[tiddler file|TiddlerFiles]] and deserialised to extract the tiddlers. Otherwise, the raw content of the file is assigned to the `text` field without any parsing
 ** ''isEditableFile'' - (optional) if `true`, changes to the tiddler be saved back to the original file. The tiddler will be saved back to the original filepath as long as it does not generate a result from the $:/config/FileSystemPath filters, which will override the final filepath generated if a result is returned from a filter. <<.from-version "5.1.23">>
+** ''searchSubdirectories'' - (optional) if `true`, all subdirectories of the //path// are searched recursively for files that match the (optional) //filesRegExp//. If no //filesRegExp// is provided, all files in all subdirectoies of the //path// are loaded. If this results is multiple files with the same tiddler title being loaded, only the last file loaded under that tiddler title will be in memory. In order to prevent this, you must have multiple directory objects listed and customize the title field with a //prefix// or //suffix// alongside the //source// attribute (see above). <<.from-version "5.1.23">>
```

Comment:
imo `<<.from-version ...` should be at the beginning of the line.

Reply:
You wrote: In order to prevent this, you **must have** multiple directory objects listed and customize 

What happens if users don't read the docs?? Will it break something. ... If not, the text should be: **you should have**

Reply:
> You wrote: In order to prevent this, you **must have** multiple directory objects listed and customize
> 
> What happens if users don't read the docs?? Will it break something. ... If not, the text should be: **you should have**

@pmario suggest re-reading, it is self-explanatory when I read it. What happens without the multiple directory objects is described in the previous sentence.

@joshuafontany typo:
`If this results is multiple files` should be `If this results in multiple files`

Reply:
@pmario @saqimtiaz Thanks guys, I have adjusted the `<<.from-version` call position and improved the docs a bit to clarify.